exporter.go

```diff
@@ -98,13 +104,19 @@ func NewGaugeContainer() *GaugeContainer {
 	}
 }
 
-func (c *GaugeContainer) Get(metricName string, labels prometheus.Labels) (prometheus.Gauge, error) {
+func (c *GaugeContainer) Get(metricName string, labels prometheus.Labels, mapping *metricMapping) (prometheus.Gauge, error) {
+	var help string
```

Comment:
Instead of copy&pasting the setting of the helptext here 4 times, I'd just ensure that the mapping is initialized correctly. Maybe by introducing a `newMapping()` function which sets help to defaultHelp.